src/modules/sensors/vehicle_imu/imu_parameters.c

```diff
@@ -48,3 +48,15 @@
 * @group Sensors
 */
 PARAM_DEFINE_INT32(IMU_INTEG_RATE, 200);
+
+/**
+ * IMU auto calibration
+ *
+ * Automatically initialize IMU (accel/gyro) calibration from bias estimates if available.
+ *
+ * @boolean
+ *
+ * @category system
+ * @group Sensors
+ */
+PARAM_DEFINE_INT32(SENS_IMU_AUTOCAL, 1);
```

Comment:
Wouldn't enabling it by default discourage people from doing the 6 sides cal?

Reply:
> Wouldn't enabling it by default discourage people from doing the 6 sides cal?

Yes in some cases, but the premise here is basically that...

1. in many typical cases this is good enough as a default start (definitely needs wider verification)
2. in most professional usage this won't apply because the product will be factory calibrated or a more diligent user will have intentionally calibrated long before the accel bias stable criteria is met (good position source)
3. if preflight accelerometer inconsistency or estimator warnings trigger the first user actionable thing will be performing a full calibration (we should review this with the events interface in mind)

Other factors to consider for the full picture.
 - it's rare that our current accels have scale factors that aren't effectively 1, and in fact I can think of a few cases where the scale factor not being 1 was an indicator of other problems (driver issues, bad accel cal for various reasons, etc)
 - the calibration offsets for our current accels are typically less than the default ekf2 accel bias learning limit
 - some users are a bit sloppy with the current accel calibration
 - some systems have a non-trivial board offset that they aren't really considering during accel cal

Reply:
Ok, I think you convinced me. I'm happy to have it enabled by default and see how it goes